### DISCUSSION TITLE
Why is the syntax key@@[default value]?

### DISCUSSION THREAD

**Discussion by juniorforlife** [robisim74/qwik-speak#62] [Q&A]
Hi, I’m wondering why the weird syntax `key@@[default value]` when using useTranslate and not just `t(“key”)`?

**Comment by robisim74** [comment]
The default value is optional.

It's useful when you want to put the translation directly into the components, and then use the [extract command](https://robisim74.gitbook.io/qwik-speak/tools/extract) to generate the files.

About the syntax `@@[default value]` directly in the key: it seems to me the fastest way to insert it, without using a new parameter in the function signature.
